irispy/sji.py

```diff
@@ -0,0 +1,159 @@
+import datetime
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+import sunpy.cm as cm
+import sunpy.instr.iris
+import sunpy.map
+import numpy as np
+import matplotlib.colors as colors
+from astropy.io import fits
+'''
+This module provides movie tools for level 2 IRIS SJI fits file
+'''
+
```

Comment:
Please add a line here defining the ```__all__``` list giving the functions to be documented online, i.e.
```__all__ = ["sji_fits_to_cube", "save_to_mp4", "dustbuster"]```